src/main/cpp/src/loader/tiledb_loader.cc

```diff
@@ -805,6 +835,9 @@ bool VCF2TileDBLoader::produce_cells_in_column_major_order(unsigned exchange_idx
   //requested in the next round or none are
   while (!m_column_major_pq.empty() && (!hit_invalid_cell || (m_column_major_pq.top())->m_column == top_column)
          && num_operators_overflow_in_this_round == 0u) {
+    if(g_show_import_progress){
```

Comment:
One other request. Just for code readability, we adhere to some rough guidelines (not as strictly as Google C++ coding style guidelines) -

```
Use spaces instead of tabs.
Use 2 spaces for indenting.
Add brackets even for one line blocks e.g. 
        if (x>0)
           do_foo();
 should ideally be 
       if (x>0) {
         do_foo();
       }
Pad header e.g.
        if(x>0) should be if (x>0)
        while(x>0) should be while (x>0)
One half indent for class modifiers.
```
Lots of existing code does not follow this, but we can slowly change that. Thanks.